### ISSUE TITLE
Error when trying to install @atomico/hooks@4

### ISSUE THREAD

**Issue by Saul-Mirone** [atomicojs/hooks#15]
To reproduce:
```bash
pnpm create vite

pnpm add @atomico/hooks
```

Result:

```bash
ERR_PNPM_SPEC_NOT_SUPPORTED_BY_ANY_RESOLVER  @atomico/use-slot@^1.0.0.1 isn't supported by any available resolver.
```

More info:

Line 58 in:
https://www.npmjs.com/package/@atomico/hooks?activeTab=code

**Comment by UpperCod** [comment]
Hi @mirone_saul , thanks for notifying this, fixed in version @atomico/hooks@4.0.1

**Comment by Saul-Mirone** [comment]
Everything works fine now.
